packages/tizen_rpc_port/example/example_proxy/lib/main.dart

```diff
@@ -0,0 +1,141 @@
+
+  @override
+  State<MyApp> createState() => _MyAppState();
+}
+
+String _message = '';
+
+class MyNotify extends NotifyCallback {
+  @override
+  Future<void> onReceived(String sender, String message) async {
+    _message = '$sender: $message';
+  }
+}
+
+class MyMessageProxy extends Message {
+  MyMessageProxy(super.appid);
+
+  String get message => _message;
+  set message(String message) => _message = message;
+
+  @override
+  Future<void> onConnected() async {
+    _message = 'Connected';
+    await register('ClientApp', MyNotify());
+  }
+
+  @override
+  Future<void> onDisconnected() async {
+    _message = 'Disconnected';
+    await connect();
+  }
+
+  @override
+  Future<void> onRejected(String errorMessage) async {
+    _message = 'Rejected error: $errorMessage';
+  }
+}
+
+class _MyAppState extends State<MyApp> {
+  late final MyMessageProxy _myProxy;
+  String _input = '';
+
+  @override
+  void initState() {
+    super.initState();
+    initPlatformState();
+  }
+
+  Future<void> initPlatformState() async {
+    try {
+      _myProxy = MyMessageProxy('com.example.tizen_rpc_port_stub_example');
+      await _myProxy.connect();
+    } on PlatformException {
+      _message = 'connect() is failed.';
+    }
+
+    if (!mounted) {
+      return;
+    }
+
+    setState(() {
+      _message = _myProxy.message;
+    });
+  }
+
+  Future<void> _sendMsg() async {
+    await _myProxy.send(_input);
+  }
+
+  Future<void> _registerCallback() async {
+    await _myProxy.register('ClientApp', MyNotify());
+    setState(() {
+      _message = 'register callback done.';
+    });
+  }
+
+  Future<void> _unregisterCallback() async {
+    await _myProxy.unregister();
+    setState(() {
+      _message = 'Unregister callback done.';
+    });
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return MaterialApp(
+      home: Scaffold(
+          appBar: AppBar(
+            title: const Text('RpcPortProxy example app'),
+          ),
+          body: SingleChildScrollView(
+            child: Column(children: <Widget>[
+              Padding(
+                padding: const EdgeInsets.fromLTRB(0.0, 50.0, 0.0, 50.0),
+                child: Text('Message: $_message\n'),
+              ),
+              TextField(
+                  onChanged: (String text) => setState(() => _input = text),
+                  decoration: const InputDecoration(
+                    border: OutlineInputBorder(),
+                    labelText: 'Input',
+                  )),
+            ]),
+          ),
+          persistentFooterButtons: <Widget>[
+            TextButton(
+              onPressed: _sendMsg,
+              child: const Text('Send'),
+            ),
+            TextButton(
+              onPressed: _registerCallback,
+              child: const Text('Register'),
+            ),
+            TextButton(
+              onPressed: _unregisterCallback,
+              child: const Text('Unregister'),
+            ),
+          ]),
```

Comment:
Proper use of trailing commas improves the readability of the code.

https://docs.flutter.dev/development/tools/formatting#using-trailing-commas

```suggestion
          ],),
```